pyproject.toml

```diff
@@ -27,6 +27,7 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.7"
 napalm = {version = "^4.0.0", optional = true}
+jsonschema = "^4.17.3"
```

Comment:
should be marked as optional right? like napalm?

Reply:
yea, in draft, still a bunch more to do, but that is one of the things.